ui/de-lib/src/main/java/org/iplantc/de/client/models/diskResources/DiskResourceMetadataList.java

```diff
@@ -13,14 +15,14 @@
 public interface DiskResourceMetadataList {
 
     @PropertyName("irods-avus")
-    void setOtherMetadata(List<DiskResourceMetadata> otherMetadata);
+    void setOtherMetadata(List<Avu> otherMetadata);
 
     @PropertyName("avus")
-    void setUserMetadata(List<DiskResourceMetadata> userMetadata);
+    void setUserMetadata(List<Avu> userMetadata);
 
     @PropertyName("irods-avus")
-    List<DiskResourceMetadata> getOtherMetadata();
+    List<Avu> getOtherMetadata();
 
     @PropertyName("avus")
-    List<DiskResourceMetadata> getUserMetadata();
+    List<Avu> getUserMetadata();
```

Comment:
So in the future, if the metadata service changes it's response format, then only `AvuList` would have to be updated to match, and not `DiskResourceMetadataList` as well.